chef/cookbooks/crowbar/recipes/crowbar-upgrade.rb

```diff
@@ -47,13 +47,14 @@
 
   bash "disable_openstack_services" do
     code <<-EOF
-      for i in $(systemctl list-units openstack* --no-legend | cut -d" " -f1) \
+      for i in $(systemctl list-units openstack* --no-legend | cut -d" " -f1 | grep -v keystone | grep -v neutron) \
                drbd.service \
                pacemaker.service;
       do
         systemctl disable $i
       done
     EOF
+    not_if { ::File.exist?("/usr/sbin/crm") }
```

Comment:
I did not say it was wrong, I'm just trying to understand why would it be needed. But I guess you can bring it back again if need.